Packs/ApiModules/Scripts/CoreIRApiModule/CoreIRApiModule.py

```diff
@@ -2077,7 +2077,7 @@ def allowlist_files_command(client, args):
     if detailed_response:
         return CommandResults(
             readable_output=tableToMarkdown('Allowlist Files', res),
-            outputs_prefix=f'{args.get("integration_context_brand", "CoreApiModule")}.blocklist',
+            outputs_prefix=f'{args.get("integration_context_brand", "CoreApiModule")}.allowlist',
```

Comment:
Isn't is suppose to be like this?
```suggestion
            outputs_prefix=f'{args.get("integration_context_brand", "CoreApiModule")}.{args.get("prefix", "allowlist")}',
```

Reply:
`detailed_response` is only in whitelist and blocklist